packages/adaptive_scaffold/example/lib/main.dart

```diff
@@ -5,6 +5,11 @@
 import 'package:adaptive_scaffold/adaptive_scaffold.dart';
 import 'package:flutter/material.dart';
 
+// A more functional demo of the usage of the adaptive layout helper widgets.
+// Modeled off of the example on the Material 3 page regarding adaptive layouts.
+// For a more clear cut example usage, please look at adaptive_layout_demo.dart
+// or adaptive_scaffold_demo.dart
+
```

Comment:
This comment is no longer needed since it is exactly the same as the docs comment below it.

Reply:
Why is this conversation resolved despite that the comment is still here?

Reply:
Serena is responding to multiple suggestions before pushing all her changes.